packages/rum-core/src/domain/rumEventsCollection/action/listenActionEvents.ts

```diff
@@ -57,6 +62,7 @@ export function listenActionEvents({ onClick }: { onClick: OnClickCallback }) {
             event: clickEvent,
             getUserActivity: () => userActivity,
           })
+          onClick = undefined
```

Comment:
We don't want to call the `onClick` callback multiple times. This can happen for example when the application is triggering a `click` event.

I'll add a test for this.

Reply:
to clarify my understanding, it is not related to the current change and could already happen, right?